driver/event_table.c

```diff
@@ -1662,7 +1662,7 @@ const struct ppm_event_info g_event_info[] = {
                                        {"cgroups", PT_BYTEBUF, PF_NA},
                                        {"env", PT_BYTEBUF, PF_NA},
                                        {"tty", PT_UINT32, PF_DEC},
-                                       {"pgid", PT_PID, PF_DEC},
+                                       {"vpgid", PT_PID, PF_DEC},
```

Comment:
this is something we never did but should be safe since the new added field takes its name, we are not removing the `pgid` field we are just moving it inside the event

Reply:
Yep i don't think we have code accessing `event->get_param(17)`, and `get_param_by_name` will still work fine.
Curious, why can't we leave `pgid` there and add `vpgid` at the end?